### PR TITLE
[WIP] Pr/toc gen

### LINKED ISSUE #87: automatically generate toc
- [x] bump jb in `requirements.txt` as soon as new jb release is out: https://github.com/executablebooks/jupyter-book/releases
- [x] remove `_toc.yml` from version control
- [x] mention `jb toc` in `CONTRIBUTING.md` 
- [x] add the `jb toc` command to the circleci config

### PR DESCRIPTION
**Pull request by prosauer**
fixes #87

### CODE REVIEW

**Comment by prosauer** [comment]
@joergbrech Generating the toc and building jb works fine now! I have naively tried to just copy the command `jupyter-book toc .` to circle CI right before `jupyter-book build -W .` and it works somehow. Unfortunately the toc generated quite a mess... even the tree-structure is not respected

**Comment by joergbrech** [comment]
Hmm, I just reran the circleci job, and now it looks fine: https://app.circleci.com/pipelines/github/joergbrech/Modellbildung-und-Simulation 

Could you add the `--add-titles` option to the toc generation command? I tried it locally on my computer. It works, but still messes with our capitalization. But if this gets fixed on the jupyter-book site and we add the option now, we don't have to change anything on our side anymore

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,8 +72,12 @@ In this section, it is assumed that you are in the root directory of your clone
  ```
  Jupyter-lab (or jupyter-notebook) runs in the browser. If your browser does not start automatically, you might have to copy the url from the command line output to your favorite browser. 
  * Create new Markdown files or jupyter notebooks or edit the ones that are already there. Check the [wiki](https://github.com/joergbrech/Modellbildung-und-Simulation/wiki) for writing conventions *(in German)*.
- * Edit `toc.yml` to include any newly created content files in the table of contents. Most likely, these are the only places where changes need to be made. For more sophisticated changes, checkout the [jupyter-book documentation](https://jupyter.org/jupyter-book/guide/01_overview) and the [demo notebook](https://jupyterbook.org/intro.html).
+ * Most likely, these are the only places where changes need to be made. For more sophisticated changes, checkout the [jupyter-book documentation](https://jupyter.org/jupyter-book/guide/01_overview) and the [demo notebook](https://jupyterbook.org/intro.html).
  * Back in the root directory, run
+  ```bash
+ jupyter-book toc content/
```
Comment (joergbrech) [inline CONTRIBUTING.md:78]: Please also add the `--add-titles` option here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,8 +72,12 @@ In this section, it is assumed that you are in the root directory of your clone
...
+ ```
+ to automatically generate the table of contents based on the page titles. Then run
  ```bash
  jupyter-book build content/
```
Comment (joergbrech) [inline CONTRIBUTING.md:82]: Could you also change this to `jupyter-book build -W content`? That is the command that is run on circleci.

**Comment by joergbrech** [comment]
jb 0.7.0 is out now. Maybe we can bump the version in `requirements.txt` again?

**Comment by joergbrech** [comment]
@prosauer, is it ok, if I take over from here? I think we should rebase on master once #105 is merged and then see if it works

**Comment by joergbrech** [comment]
Closing thins in favor of #110

**Pull request closed without merging**